edb/server/protocol/binary.pyx

```diff
@@ -1153,30 +1153,30 @@ cdef class EdgeConnection:
     def signal_side_effects(self, side_effects):
         if side_effects & dbview.SideEffects.SchemaChanges:
             self.server.create_task(
```

Comment:
To make it less ambiguous perhaps we should call this `schedule_task`. But please allow me to think more about this. This can be fixed in rc3 too.